Mathlib/LinearAlgebra/Multilinear/Basic.lean

```diff
@@ -252,6 +254,21 @@ def toLinearMap [DecidableEq ι] (m : ∀ i, M₁ i) (i : ι) : M₁ i →ₗ[R]
 #align multilinear_map.to_linear_map MultilinearMap.toLinearMap
 #align multilinear_map.to_linear_map_to_add_hom_apply MultilinearMap.toLinearMap_apply
 
+/-- This introduces the "derivative" of a multilinear map, as a linear map from
+`(i : ι) → M₁` to `M₂`. For continuous multilinear maps, this will indeed be the
+derivative.-/
+noncomputable def linearDeriv [DecidableEq ι] [Fintype ι] (f : MultilinearMap R M₁ M₂)
```

Comment:
Why is this noncomputable?

Reply:
Because Lean originally complained about it. But I tried removing the "noncomputable" and it's fine.